Xamarin.Forms.Platform.Android/Renderers/ShellToolbarAppearanceTracker.cs

```diff
@@ -36,10 +37,23 @@ protected virtual void SetColors(Toolbar toolbar, IShellToolbarTracker toolbarTr
 		{
 			var titleArgb = title.ToAndroid(ShellRenderer.DefaultTitleColor).ToArgb();
 
-			toolbar.SetTitleTextColor(titleArgb);
-			using (var colorDrawable = new ColorDrawable(background.ToAndroid(ShellRenderer.DefaultBackgroundColor)))
-				toolbar.SetBackground(colorDrawable);
-			toolbarTracker.TintColor = foreground.IsDefault ? ShellRenderer.DefaultForegroundColor : foreground;
+			if (_titleTextColor != titleArgb)
```

Comment:
Setting these values was causing a slight flickering of the hamburger icon when you popped a page